invenio_records_lom/resources/serializers/oai/schema.py

```diff
@@ -221,11 +221,12 @@ def get_centity(self, contributors: dict) -> list:
 class LifecycleSchema(ExcludeUnknownOrderedSchema):
     """Schema for LOM-UIBK's `lifecycle` category."""
 
+    datetime = fields.String()
     version = fields.Field()
     status = fields.Field()
     contribute = fields.List(fields.Nested(ContributeSchema()))
 
-    @pre_load
+    @pre_dump
```

Comment:
that seems a strange change. `@pre_load` is something completely different to `@pre_dump` or am i wrong. the `group_contributes_by_role` will not called anymore on the import process. this would mean that the information will not be in our database and only added for the oai-pmh (and other exports)

Reply:
`@pre_load` was an error to begin with
it should always have been `@pre_dump`

see updated PR-message for more context